docs/getting_started.md

```diff
@@ -132,36 +132,27 @@ Experiment with the graph range parameters and other settings.
 Let us make this more interesting and start some example targets for Prometheus
 to scrape.
 
-The Go client library includes an example which exports fictional RPC latencies
-for three services with different latency distributions.
-
-Ensure you have the [Go compiler installed](https://golang.org/doc/install) and
-have a [working Go build environment](https://golang.org/doc/code.html) (with
-correct `GOPATH`) set up.
-
-Download the Go client library for Prometheus and run three of these example
-processes:
+The next steps are for `node_exporter` to install follow
+[these steps.](https://prometheus.io/docs/guides/node-exporter/)
 
 ```bash
-# Fetch the client library code and compile example.
-git clone https://github.com/prometheus/client_golang.git
-cd client_golang/examples/random
-go get -d
-go build
+# <version> and <target> will depend on what you downloaded.
+tar -xzvf node_exporter-<version>.<target>.tar.gz
```

Comment:
You can do * instead of the pseudo-variables, as long as only one version is downloaded it'll do the right thing and is easier for users.